tests/yz_crdt.erl

```diff
@@ -113,6 +127,362 @@ confirm() ->
 
     pass.
 
+test_repeat_sets(Pid, Cluster, Bucket, Index, Key) ->
```

Comment:
sure. this has been waiting around in purgatory since the customer days... so that would be why it lacks it a bit. will be done shortly.